test/core/markets/pieceio/CMakeLists.txt

```diff
@@ -11,6 +11,3 @@ target_link_libraries(pieceio_test
     unixfs
     ipfs_datastore_in_memory
     )
-target_include_directories(pieceio_test PUBLIC
```

Comment:
make pieceio_test fails: fatal error: 'testutil/resources/resources.hpp' file not found
Since resources are generated in "${TEST_RESOURCES_PATH}" directory.

Reply:
Works on my machine and on CI